sql_scripts/milestone_3_ddl_commands/create_tables.sql

```diff
@@ -0,0 +1,56 @@
+-- Active: 1695541481884@@127.0.0.1@3306@Dhan_Kharch
+CREATE TABLE UserRoles (
+  role_ID INT PRIMARY KEY UNIQUE AUTO_INCREMENT,
+  role_name ENUM('admin', 'user')
+);
+
+CREATE TABLE Users (
+  user_ID INT PRIMARY KEY,
+  username VARCHAR(255),
+  email_id VARCHAR(255) UNIQUE,
+  role_ID INT,
+  FOREIGN KEY (role_ID) REFERENCES UserRoles(role_ID)
+);
+
+CREATE TABLE Accounts (
```

Comment:
Two questions:

- if I have an account like credit card, or any other prepaid card, can your system accommodate those types of accounts?
- What i don't want an account to be included in the budget, does your system facilitate that?